core/domain/question_fetchers_test.py

```diff
@@ -132,13 +152,17 @@ def test_cannot_get_question_from_model_with_invalid_schema_version(self):
         self.assertEqual(all_question_models.count(), 1)
         question_model = all_question_models.get()
```

Comment:
You can maybe try renaming this to avoid the error mentioned above.

Reply:
Done